.deliveroo-local/.env

```diff
@@ -0,0 +1,10 @@
+PORT=3000
+RACK_ENV=development
+ROUTEMASTER_CACHE_VERIFY_SSL=false
+ROUTEMASTER_CLIENTS=demo,identity-service,late-order-notifier,logistics-dashboard,orderweb
+ROUTEMASTER_DRAIN_TOKENS=demo,identity-service,late-order-notifier,logistics-dashboard,orderweb
+ROUTEMASTER_LOG_LEVEL=INFO
+ROUTEMASTER_REDIS_URL=redis://redis:6379/0
+ROUTEMASTER_ROOT_KEY=routemaster
+ROUTEMASTER_SSL_VERIFY=NO
+ROUTEMASTER_URL=https://routemaster.deliveroo-local.com
```

Comment:
Is this safe?

Reply:
Can you elaborate?

Reply:
Sorry for the lack of detail, I realise that this is a public repository.

I'm going to talk with @deliveroo/infrasec (who setup the domain) just to check a few things.  My specific worry is that the `-local` suggests that this will be using RFC1918 addresses on a domain in the public DNS namespace.